easybuild/framework/easyblock.py

```diff
@@ -56,15 +56,16 @@
 from easybuild.framework.easyconfig.easyconfig import get_easyblock_class, get_module_path, resolve_template
 from easybuild.framework.easyconfig.tools import get_paths_for
 from easybuild.framework.easyconfig.templates import TEMPLATE_NAMES_EASYBLOCK_RUN_STEP
+from easybuild.tools.downloadtools import PROTOCOL_MAP
```

Comment:
keep import lines alphabetical, that will avoid you downloading from the same module twice, as you're doing here :)